src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -93,6 +94,8 @@ public Http2Stream(HttpRequestMessage request, Http2Connection connection, int s
 
                 _headerBudgetRemaining = connection._pool.Settings._maxResponseHeadersLength * 1024;
 
+                _canRetry = false;
```

Comment:
This isn't necessary; it defaults to false.

Reply:
removed